lib/dynamoid/config.rb

```diff
@@ -50,6 +50,7 @@ module Config
     option :store_date_as_string, default: false # store Date fields in ISO 8601 string format
     option :store_empty_string_as_nil, default: true # store attribute's empty String value as null
     option :store_boolean_as_native, default: true
+    option :store_binary_as_native, default: false
```

Comment:
Yeah, it makes sense to have also a global config option. But it doesn't affect field conversion and isn't tested yet.